data/low_income_thresholds.json

```diff
@@ -320,5 +320,33 @@
         "8": 130300
       }
     }
+  },
+  "WI": {
+    "default": {
+      "source_url": "https://focusonenergy.com/income-qualified",
+      "thresholds": {
+        "1": 66300,
+        "2": 75750,
+        "3": 85200,
+        "4": 94650,
+        "5": 102250,
+        "6": 109800,
+        "7": 117400,
+        "8": 124950
+      }
+    },
+    "wi-focus-on-energy": {
```

Comment:
This is a bug/awkward spot with the tooling right now where it is much easier/more automated to define a dummy default that is never used than to use the default.

For that reason, I'm fine leaving this as is, though other reviewers may ask you to remove the wi-focus-on-energy entry.